bin/sozo/src/ops/migration/mod.rs

```diff
@@ -306,12 +312,19 @@ where
     ui.print_step(1, "🌎", "Building World state...");
 
     let mut local_manifest =
-        BaseManifest::load_from_path(&manifest_dir.join(MANIFESTS_DIR).join(BASE_DIR))?;
+        match BaseManifest::load_from_path(&manifest_dir.join(MANIFESTS_DIR).join(BASE_DIR)) {
+            Ok(manifest) => manifest,
+            _ => return Err(anyhow!("Fail to load local manifest file.")),
+        };
```

Comment:
```suggestion
         BaseManifest::load_from_path(&manifest_dir.join(MANIFESTS_DIR).join(BASE_DIR))
         .map_err(|_| anyhow!("Fail to load local manifest file."))?;
```
`map_err` may be more compact and idiomatic here.

Reply:
Yes thanks ! I still have to learn this kind of idiomatic pattern `map_err`, `map_or` ... :-)